gossip/minimal/goshawk.go

```diff
@@ -173,30 +156,28 @@ func (hawk *Goshawk) Fly(ctx context.Context) {
 			glog.Infof("finished Checker(%s)", origin.Name)
 		}(origin)
 	}
+
 	wg.Wait()
 }
 
 // Scanner runs a continuous scan of the destination log.
 func (hawk *Goshawk) Scanner(ctx context.Context) {
```

Comment:
nit: Could the method name be a verb, e.g. `Scan`?

Reply:
No, the name is deliberate and consistent -- all of the methods that do something continuously or continually (and so get run in goroutines) have names that are nouns built from a verb-subject -- `Checker`, `Retriever`, `Scanner`, `Submitter`.

Reply:
Not sure I support this, unless this rule is documented somewhere. Standard Go libs don't seem to follow this convention. For example, `http.Server`'s continuous/blocking "run" method is still a verb: https://golang.org/pkg/net/http/#Server.ListenAndServe.

Reply:
I believe Go encourages "*er"  suffixes in interfaces.  ie. `io.Writer` and `io.Reader`.